mubench.reviewsite/templates/detector.phtml

```diff
@@ -117,6 +117,40 @@ use MuBench\ReviewSite\Model\ReviewState;
                             <i class="fa fa-thumbs-o-down" aria-hidden="true" style="color:darkred;" title="Resolved: No"></i>
                             (<a href="<?= $review_url ?>/resolution"><i class="fa fa-handshake-o" aria-hidden="true" title="Resolution"></i></a>)
                         <?php endif; ?>
+                        <?php if($user && $misuse->hasReviewed($user)): ?>
+                            <?php
+                                $user_review = $misuse->getReview($user);
+                                $reviews = $misuse->getReviews();
+                            ?>
+                            <?php if($review_state == ReviewState::NEEDS_REVIEW && $user_review->hasPositiveHit()) : ?>
```

Comment:
The condition `$user_review->hasPositiveHit()` should be true for all subsequent cases, i.e., if the user did not identify a hit, we never want to show no rank. I know that the subsequent cases, by checking for an agreement, implicitly also check that the user identified a hit. However, I'd prefer this check to wrap this entire thing, just like the `hasReviewed($user)`.